app/Http/Controllers/RegisteredUserController.php

```diff
@@ -6,27 +6,76 @@
 use Illuminate\Support\Facades\Auth;
 use App\Models\User;
 use Illuminate\Validation\Rules\Password;
+use Illuminate\Validation\ValidationException;
 
 class RegisteredUserController extends Controller
 {
     public function create()
     {
-        return view('register');
+        return view('user.create');
     }
 
-    public function store()
+    public function show(Request $request)
     {
-        $validatedAttributes = request()->validate([
-            'name' => ['required'],
+        $isHTMX = $request->hasHeader('HX-Request');
+        return view('user.show', ['isHTMX' => $isHTMX, 'user' => auth()->user()])->fragmentIf($isHTMX, 'show-user');
+    }
+
+    public function store(Request $request)
+    {
+        $data = $request->validate([
+            'name' => ['required', 'unique:users,name'],
```

Comment:
Not sure that this is needed, but this edge case unlikely to be encountered by our team.